pad/hori/Fighting Stick α/config.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
I’d recommend not using anything other than the following characters for folder/file names:
```
0123456789
abcdefghijklmnopqrstuvwxyz
_
```
Can you rename this controller’s folder to something like `fightstick_alpha` please?